server/grpc_service.go

```diff
@@ -1960,10 +2026,25 @@ func (s *GrpcServer) GetOperator(ctx context.Context, request *pdpb.GetOperatorR
 }
 
 // validateRequest checks if Server is leader and clusterID is matched.
-// TODO: Call it in gRPC interceptor.
 func (s *GrpcServer) validateRequest(header *pdpb.RequestHeader) error {
-	if s.IsClosed() || !s.member.IsLeader() {
-		return ErrNotLeader
+	return s.validateRoleInRequest(context.TODO(), header, nil)
```

Comment:
Why not just name it as validateRequest?

Reply:
Because some code needs original `validateRequest` which is only one param and is easy to use, so I keep it.